helm-charts/orchestrator/values.yaml

```diff
@@ -23,6 +23,7 @@ rhdhOperator:
   enabled: true # whether the operator should be deployed by the chart
   enableGuestProvider: false # whether to enable guest provider
   catalogBranch: v1.3.x # The branch for https://github.com/parodos-dev/workflow-software-templates used to import software templates resources
+  existingRHDHNamespace: # Namespace for Existing RHDH Instance
```

Comment:
should the default value for this property be an empty string ("") instead of no value?
seems to be a convention for fields without a value in this file.

Reply:
Ah! my bad. That should definitely be an empty string. Fixed it [here](https://github.com/parodos-dev/orchestrator-helm-operator/pull/320/commits/5cbc55910038658bd175d5fde58c3bb5e7c40a50). thanks for the catch.

Reply:
the only missing piece which I'm not sure is available to the user outside of this github repository is the update for https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/helm-charts/orchestrator/README.md
it used to be generated by https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/Makefile#L9 in the previous repo, but since we're still maintaining this README.md file in this repo, we should keep it up to date.